cluster-autoscaler/core/static_autoscaler.go

```diff
@@ -57,6 +57,9 @@ const (
 	unschedulablePodWithGpuTimeBuffer = 30 * time.Second
 	// How long should Cluster Autoscaler wait for nodes to become ready after start.
 	nodesNotReadyAfterStartTimeout = 10 * time.Minute
+
+	// NodeUpcomingAnnotation is an annotation CA adds to nodes which are upcoming.
+	NodeUpcomingAnnotation = "cloud.google.com/cluster-autoscaler-node-upcoming"
```

Comment:
This is in no way specific to GCP. Please use a more generic CA prefix.

Reply:
My bad, changed!